.github/workflows/python.yml

```diff
@@ -63,17 +63,16 @@ jobs:
           virtualenv "$venv"
           source "$venv/bin/activate"
           pip install mypy
-          mypy algorithms/
+          mypy algorithms/ .
```

Comment:
You are running `mypy` on two directories:
* `algorithms/`
* `.` (the current directory) which also happens to contain `algorithms`

So you are running `mypy` on `algorithms` twice.

Reply:
"." has been removed